src/jit/compiler.h

```diff
@@ -7984,7 +7984,9 @@ public :
         // Methods returning a struct in two registers is considered having a return value of TYP_STRUCT.
         // Such method's compRetNativeType is TYP_STRUCT without a hidden RetBufArg
         return varTypeIsStruct(info.compRetNativeType) && (info.compRetBuffArg == BAD_VAR_NUM);
-#else 
+#elif FEATURE_MULTIREG_RET && defined(_TARGET_X86_)
+        return varTypeIsLong(info.compRetNativeType);
```

Comment:
A short comment here stating (the obvious) that x86 returns long in two registers would be good.